packages/valory/skills/mech_interact_abci/behaviours/request.py

```diff
@@ -310,6 +327,7 @@ def async_act(self) -> Generator:
                 )
```

Comment:
This logging should not be present or at least not at an `info` level. Moreover, `serialized_data` is a generator.

Reply:
Please make a concrete suggestion how to fix this.

Reply:
Opened #153.